src/components/table/LightTable.tsx

```diff
@@ -241,38 +261,54 @@ function LightTable({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [searchString, page, rowsPerPage, orderProperty, orderDirection, tableName]);
 
+  const supportsResize = columns.some((col) => col.allowResize);
+  const tableWidth = columns.reduce((agg, column) => agg + (column.width || NaN), 0);
+  const effectiveTableWidth = _.isNaN(tableWidth) || !supportsResize ? '100%' : tableWidth;
   return (
     <div>
       {!loading && (
         <Paper className={classes.root}>
-          <div className={classes.tableWrapper}>
-            <Table className={classes.table}>
-              <LightTableHead columns={columns} onRequestSort={handleRequestSort} />
+          <TableContainer className={classes.tableWrapper} sx={{ maxHeight: 300 }}>
```

Comment:
Intentional change of the maxHeight?  If the tableWrapper class is being applied to the TableContainer, why not just change the maxHeight there?

Reply:
derp...no, my bad.  Left over from trying to get the table to behave.  Will remove

Reply:
<img width="321" alt="Screen Shot 2022-09-06 at 10 20 54 AM" src="https://user-images.githubusercontent.com/111771148/188659979-6300dac1-b8d2-4c29-82c5-67169bada1f3.png">

Reply:
right exactly. The class takes care of it.